tests/func/experiments/test_experiments.py

```diff
@@ -399,9 +418,7 @@ def test_untracked(tmp_dir, scm, dvc, caplog, workspace):
 
     # copy.py is untracked
     with caplog.at_level(logging.ERROR):
-        results = dvc.experiments.run(
-            stage.addressing, params=["foo=2"], tmp_dir=True
-        )
+        results = dvc.experiments.run(stage.addressing, tmp_dir=True)
```

Comment:
This needs careful review to make sure I am not invalidating this test.

I don't believe this call to exp run needs to be modifying the parameter as there appears to be no params file at the time of running (thus throws the new error unexpectedly) and the file is checked afterwards, as it is in the second exp run in the test. But there could be something to the test that I am not understanding.

Reply:
This test is checking for the behavior that happens when any important files are untracked (completely new file from git's perspective). In this case, with the new behavior we end up hitting the error for having a new parameter first (before we get to the relevant checks for the untracked files behavior).

It would be better to revert this change and also generate `params.yaml` as a git-tracked file from the start (so that the params existence is irrelevant for this test).

So these changes should be reverted, and line 409 should become:

```py
    tmp_dir.scm_gen("params.yaml", "foo: 1", commit="track params")
```